tests/ui/invalid_need_module_arg_position.rs

```diff
@@ -3,10 +3,10 @@ use pyo3::prelude::*;
 #[pymodule]
 fn module(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
     #[pyfn(m, pass_module)]
-    fn fail(string: &str, module: &PyModule) -> PyResult<&str> {
+    fn fail<'py>(string: &str, module: &'py PyModule) -> PyResult<&'py str> {
```

Comment:
The original check was masking an ambiguous lifetime error; I needed to add lifetime hints to avoid having this error also included in the trybuild output.